install-local.sh

```diff
@@ -174,18 +174,18 @@ if [ "$ns_pyvenv" != disable ]; then
     msghi "Initializing python virtual environment"
     ns_pyvenv_opt=
     if [ "$ns_pyvenv" == inherit ]; then
-	ns_pyvenv_opt=--system-site-packages
+    ns_pyvenv_opt=--system-site-packages
```

Comment:
The old indentation is correct here... at least in magntitude; i.e., eight spaces, not four.